### DISCUSSION TITLE
No output shown by process

### DISCUSSION THREAD

**Discussion by saint7007** [nextflow-io/nextflow#3679] [Q&A]
Hi all,
Can anyone help me with this,
![image](https://user-images.githubusercontent.com/10279812/220945715-f1afacd4-de85-4192-ab62-0b98c36399b9.png)

It runs but there is no status on what is going on with script.

**Comment by pditommaso** [comment]
Is this the screenshot of your mobile phone?

**Reply by saint7007** [reply]
sorry, some issue while blurring user name of server, will rectify.

**Reply by saint7007** [reply]
@pditommaso can you check now?

**Comment by mribeirodantas** [comment]
Hello, @saint7007. Could you please share a minimal reproducible example? 😄
